src/client/VZSidebar/CreateFileModal.tsx

```diff
@@ -30,6 +30,13 @@ export const CreateFileModal = ({
     setNewName('');
   }, [newName, onRename]);
 
+  //returns true if filename isvalid, false otherwise
+  const validateFileName = useCallback((fileName : string) => {
+    let regex = /^[a-zA-Z0-9](?:[a-zA-Z0-9 ./+=_-]*[a-zA-Z0-9])?$/;
```

Comment:
```suggestion
    const regex = /^[a-zA-Z0-9](?:[a-zA-Z0-9 ./+=_-]*[a-zA-Z0-9])?$/;
```

If this was from source, it's best to include the link to where it was found.

Also, it would be good to list out in a comment what is meant by "valid".